themes/powerlevel10k_modern.opm.json

```diff
@@ -0,0 +1,122 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh3/main/themes/schema.json",
+    "blocks": [
+      {
+        "type": "prompt",
+        "alignment": "left",
+        "segments": [
+          {
+            "type": "os",
+            "style": "diamond",
+            "foreground": "#000000",
+            "background": "#ffffff",
+            "leading_diamond": "\uE0B6",
+            "trailing_diamond": "\uE0B4",
+            "properties": {
+            }
+          },
+          {
+            "type": "root",
+            "style": "powerline",
+            "foreground": "#000000",
+            "background": "#0000ff",
+            "powerline_symbol": "\uE0B4",
+            "properties": {
+              "style": "full"
+            }
+          },
+          {
+            "type": "path",
+            "style": "powerline",
+            "foreground": "#ffffff",
+            "background": "#0000ff",
+            "powerline_symbol": "\uE0B4",
+            "properties": {
+              "style": "full",
+              "postfix": " "
+            }
+          },
+          {
+            "type": "git",
+            "style": "powerline",
+            "foreground": "#000000",
+            "background": "#D4E157",
+            "powerline_symbol": "\uE0B4",
+            "properties": {
+              "style": "full",
+              "prefix": " "
+            }
+          },
+          {
+            "type": "text",
+            "style": "plain",
+            "foreground": "#546E7A",
+            "background": "transparent",
+            "properties": {
+              "text": "",
```

Comment:
What's the use-case of this segment when the text isn't there?

Reply:
None, Seems like i forgot to remove this segment.